src/components/addNew.js

```diff
@@ -22,26 +24,37 @@ const AddNew = () => {
     const id = uuidv4();
     const { title, category } = inputValues;
 
-    const newBook = {
-      item_id: id,
-      title,
-      category,
-    };
-    dispatch(addBook(newBook));
+    if (title.trim()) {
+      if (category.trim()) {
+        const newBook = {
+          item_id: id,
+          title,
+          category,
+        };
+        dispatch(addBook(newBook));
+      } else {
+        setInputValues({
+          errMsg: 'Please select category',
+        });
+      }
+    } else {
+      setInputValues({
+        errMsg: 'Please Add Title',
+      });
+    }
   };
```

Comment:
- [ ] When I clicked on the `add new book` without filling out the title and category, I got this error. please fix it.

![setup-11](https://user-images.githubusercontent.com/55635977/145484992-6ac5ba65-6d26-4089-9e31-18c82a51452e.png)